automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestFunctions.scala

```diff
@@ -34,16 +33,61 @@ class TestFunctions extends ExtractionTest {
     testFunctionEvent(mentions, desired)
   }
 
+  val t4a = "The actual vapor pressure can be measured or it can be calculated from various humidity data, such as measured dew point temperature, wet-bulb and dry-bulb temperature, or relative humidity and air temperature data."
+  failingTest should s"find functions from t4a: ${t4a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "The actual vapor pressure" -> Seq("various humidity data") // todo: how can we deal with pronouns (i.e. it)? + another issue is the "such as" part. (needs discourse parsing?)
```

Comment:
Based on this example, this heuristic from Dane's paper looks correct

> a generally trustworthy heuristic that theearliest named entity in the sentence is likely to be the an-tecedent of a pronoun if they match grammatically (Hobbs,1978).  (An Investigation of Coreference Phenomena in the Biomedical Domain, Bell et al, 2016)

This does not have to be in this PR, though.